translator/internaldata/oc_to_metrics.go

```diff
@@ -229,7 +235,7 @@ func fillLabelsMap(ocLabelsKeys []*ocmetrics.LabelKey, ocLabelValues []*ocmetric
 	}
 }
 
-func fillIntDataPoint(ocMetric *ocmetrics.Metric, dps pdata.IntDataPointSlice) {
+func fillNumberDataPointAsInt(ocMetric *ocmetrics.Metric, dps pdata.NumberDataPointSlice) {
```

Comment:
This might be a good opportunity to pass `valType` into this function and remove the need for two separate functions that are almost identical.

Reply:
Already reduced duplicate code, if I pass value type I need to pass the value as interface and convert. Prefer type safety.

Reply:
Good point